ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -132,7 +132,7 @@ tests:
   steps:
     cluster_profile: aws
     env:
-      FEATURE_GATES: '[GatewayAPI=true,ClusterAPIInstallAWS=true]'
+      FEATURE_GATES: '[GatewayAPI=true,GatewayAPIController=true,ClusterAPIInstallAWS=true]'
```

Comment:
`ClusterAPIInstallAWS=true` could be removed, it was added in 4.16 to temporarily workaround Cluster API install issue on AWS.

Reply:
I can do it in another PR, to avoid any impact on the ongoing development.

Reply:
I think that Hongan is saying that enabling `ClusterAPIInstallAWS` is a no-op, so keeping or removing it wouldn't impact ongoing development one way or the other.  Removing it would clean up some old cruft.  Do you still prefer to leave it for a separate PR?